generators/ci-cd/templates/_.gitlab-ci.yml

```diff
@@ -1,21 +1,19 @@
-<%_ if (integrations.includes('docker')) { _%>
```

Comment:
Probably, it can be done in generator, but immediatly, I don't know how :stuck_out_tongue: 
But you can launch the ci-cd sub gen twice:
- 1st time for Jenkins without Docker
- 2nd time for gitlab with Docker
It should work, right ?